vlsi/Makefile

```diff
@@ -18,22 +18,30 @@ include $(base_dir)/variables.mk
 # vlsi types and rules
 #########################################################################################
 sim_name           ?= vcs # needed for GenerateSimFiles, but is unused
-tech_name          ?=
-tech_dir           ?= $(vlsi_dir)/hammer-$(tech_name)-plugin/$(tech_name)
+tech_name          ?= asap7
+tech_dir           ?= $(if $(filter $(tech_name), asap7), $(vlsi_dir)/hammer/src/hammer-vlsi/technology/$(tech_name), $(vlsi_dir)/hammer-$(tech_name)-plugin/$(tech_name))
 SMEMS_COMP         ?= $(tech_dir)/sram-compiler.json
 SMEMS_CACHE        ?= $(tech_dir)/sram-cache.json
 SMEMS_HAMMER       ?= $(build_dir)/$(long_name).mems.hammer.json
 MACROCOMPILER_MODE ?= -l $(SMEMS_CACHE) -hir $(SMEMS_HAMMER)
 OBJ_DIR            ?= $(vlsi_dir)/build
-ENV_YML            ?= $(vlsi_dir)/bwrc-env.yml
-INPUT_CONFS        ?= example.yml $(dir $(tech_dir))/bwrc.yml
+ENV_YML            ?= $(vlsi_dir)/env.yml
+INPUT_CONFS        ?= example.yml
 HAMMER_EXEC        ?= ./example-vlsi
+VLSI_TOP           ?= $(TOP)
```

Comment:
Out of curiosity, is there a reason we need a separate `VLSI_TOP` variable? IIRC specifying `make TOP=MyBlock` should do the right thing. 

Its fine if we resolve this in a separate PR.

Reply:
You can't set `TOP` to a top_module that's not described in the Chisel you're trying to elaborate.

Reply:
(if this comes up as a questions, it's worth writing it explicitly in the docs :) )